packages/site-landing/src/require-shim.d.ts

```diff
@@ -1,14 +0,0 @@
-/**
- * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
- * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
- * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
- * and https://github.com/palantir/blueprint/blob/master/PATENTS
- */
-
-// borrowed from https://github.com/TypeStrong/ts-loader
-
-declare var require: {
```

Comment:
awesome that we can delete this! where does the type info come from now?

Reply:
`@types/node` at the root